lib/GridListImageItem/GridListImageItem.js

```diff
@@ -191,6 +191,7 @@ module.exports = kind(
 	* @private
 	*/
 	imageSizingChanged: function () {
+		this.$.image.addRemoveClass('enyo-gridlist-imageitem-image', (!this.imageSizing && !!this.placeholder));
```

Comment:
When we are using `imageSizing`, the `sized-image` class effectively adds the 100% width and height declarations on the image, so I am not sure why we need to make this conditional on *not* having `imageSizing` specified.